server/WebAPI/Controllers/AccommodationsController.cs

```diff
@@ -40,6 +44,13 @@ public IActionResult GetAccommodation([FromRoute] int id)
       return Ok(accommodation);
     }
 
+    [HttpGet]
+    [Route(("max-price"))]
```

Comment:
why the double brackets?(())

you can remove [route] and change [httpget] to [httpget ("max-price")], if you wish:)